### ISSUE TITLE
Issue building sample solution/projects

### ISSUE THREAD

**Issue by KinsonDigital** [dotnet/maui-samples#114]
## Issue:

I am having issues with building anything in the solution for the sample projects.

## Reproduce Steps:

1. Checkout tag **6.0.1xx-preview2**
2. Open solution with Visual Studio Preview
3. Build Solution
4. Observe the results below:

**Errors In Error List:**
![image](https://user-images.githubusercontent.com/4921022/112668611-187a2100-8e35-11eb-8d2d-059c84e1d1ba.png)

**Error Message:**
![image](https://user-images.githubusercontent.com/4921022/112668660-26c83d00-8e35-11eb-86ea-3d227eda55cb.png)

---

## Other Info:

**Visual Studio Version:** v16.10.0 Preview v1.0
**.NET SDK:** Installed .NET SDK 6.0.100-preview.2.21155.3(x64)
**Windows Version:** v20H2 (OS Build 19042.868)

**Comment by jonathanpeppers** [comment]
There is a manual step to create `EnableWorkloadResolver.sentinel`, did you do that part already?

https://github.com/dotnet/net6-mobile-samples#using-ides

With this change in place, the HelloMaui project won't work inside Visual Studio yet, but you can unload it to try HelloAndroid and HelloiOS.

**Comment by KinsonDigital** [comment]
I did not try attempt that part yet.

Let me give that a shot.  😊

**Comment by KinsonDigital** [comment]
When running the command from an admin command prompt, it did not work and said **Access Denied**.  This was with all instances of VS shutdown.

I tried the powershell version and it ran with no issues.

I started up VS preview and opened the solution, unloaded the **HelloMaui** project and rebuilt the solution and I am getting the below results.

![image](https://user-images.githubusercontent.com/4921022/112684931-30a86b00-8e4a-11eb-96c2-e87d82f322cf.png)
